### DISCUSSION TITLE
No support for digest type authentication

### DISCUSSION THREAD

**Discussion by schrill** [bitfireAT/icsx5#259] [General]
Application uses basic authentication when entering URL and after which username and pass.
Error output from server:
[Sat Mar 23 19:21:54.499669 2024] [auth_digest:error] [pid 4077402] [client -:34472] AH01781: client used wrong authentication scheme `Basic': /webdav/calendar/calendar.ics

**Comment by sunkup** [comment]
Please take a look at this: https://github.com/bitfireAT/davx5-ose/discussions/676#discussioncomment-8912339

I would not add support for digest type auth, for the same reason. That being

> However Digest auth shouldn't be used because it adds complexity for no advantage. Basic auth with HTTPS is as secure as digest auth with HTTPS. What is your use-case for Digest?
